api/tests/Behat/bootstrap/Common/AuthenticationTrait.php

```diff
@@ -46,15 +46,15 @@ public function iAmAtAdminLogin()
      */
     public function openActivationOrPasswordResetPage($admin, $pageType, $email)
     {
-        $url = sprintf('/admin/fixture/user-registration-token?email=%s', $email);
+        $url = sprintf('/admin/fixtures/user-registration-token?email=%s', $email);
         $this->iAmLoggedInToAdminAsWithPassword('admin@publicguardian.gov.uk', 'DigidepsPass1234');
         $this->visitAdminPath($url);
         $token = $this->getSession()->getPage()->getContent();
         $this->visitAdminPath('/logout');
 
-        $page = $pageType === 'activation' ? 'activate' : 'password-reset';
+        $page = 'activation' === $pageType ? 'activate' : 'password-reset';
 
-        if ($admin === '' || $admin === false) {
+        if ('' === $admin || false === $admin) {
```

Comment:
I know this is the auto formatting but I find this totally not how my brain works hehe.